typescript/sdk/src/contracts.ts

```diff
@@ -112,3 +113,21 @@ export function connectContractsMap<F extends HyperlaneFactories>(
     connectContracts(contracts, multiProvider.getSignerOrProvider(chain)),
   );
 }
+
+export async function filterOwnableContracts(
+  contracts: HyperlaneContracts<any>,
+): Promise<{ [key: string]: Ownable }> {
+  const isOwnable = async (_: string, contract: Contract): Promise<boolean> => {
```

Comment:
Nit, don't see a reason for this function to be an inline lambda instead of a plain ol function